tutorials/using-stackdriver-uptime-checks-for-scheduling-cloud-functions/index.md

```diff
@@ -0,0 +1,518 @@
+        var now = new Date().toUTCString();
+        console.log(`GET received at ${now}`);
+        res.status(200).end();
+    }
+    
+    function handlePOST(req, res) {
+        var now = new Date().toUTCString();
+        console.log(`POST received at ${now} with ${req.body.data}`);
+        res.status(200).end();
+    }
+    ```
+    
+This function can be triggered using HTTP GET or POST requests. In each case, it responds with a message showing the HTTP method used to trigger it, the time it was triggered, and any data associated with POST requests. For all other HTTP methods, the function returns a 405 HTTP status code.
+
+After a successful execution, the function logs a message to the console with the HTTP method and the timestamp of the request. These console logs appear in the Cloud Function's logs and can be queried using the `gcloud` tool.  They can be used to verify whether the function was executed on schedule.
+
+Because the function handles GET and POST requests differently you’re able to simulate different Cloud Functions for the trigger list simply by invoking it in different ways.
+
+# Deploying the function
+
+1.  Run the following command to deploy the function:
+    ```
+    gcloud beta functions deploy cronTarget --stage-bucket ${STAGE_BUCKET} --trigger-http
+    ```
+    The `--trigger-http` option generates and assigns a URL endpoint to the function so that it can be triggered with an HTTP request to that endpoint.
+    
+    It might take a few minutes for the command to finish running. After deployment is finished, you’ll see something like the following:
+    <pre>
+    ...
+    Deploying function (may take a while - up to 2 minutes)...done.
+    availableMemoryMb: 256
+    entryPoint: cronTarget
+    httpsTrigger:
+      url: <b>https://us-central1-your-project.cloudfunctions.net/cronTarget</b>
+    latestOperation: operations/Z2NmLXNlY3VyZS91cy1jZW50cmFsMS9zZWN1cmUvT0FYclM0N2ttRmc
+    ...
+    </pre>
+    Note the URL in bold. This is the URL that you’ll use to trigger the Cloud Function. From this point on, the triggering URL is referred to as **`[TARGET_FUNCTION_URL]`**.
+    
+1.  Verify the status of the deployment by running the following command:
+    ```
+    gcloud beta functions describe cronTarget
+    ```
+    This command describes the configuration and current status of the named Cloud Function, in this case `cronTarget`.
+    
+    The command outputs something like the following:
+    ```
+    ...
+    status: READY
+    timeout: 60s
+    ...
+    ```
+    A `READY` status indicates that the function has been successfully deployed and is ready to be invoked.
+
+# Creating the trigger list
+
+When the master Cloud Function is executed it invokes the Cloud Functions specified by the trigger list. The trigger list contains a list of URLs corresponding to target HTTP Cloud Functions and the information required to trigger them, such as the HTTP method and any POST data.
```

Comment:
The master Cloud Function invokes the Cloud Functions that are specified by the trigger list.